Aptoide/bds-sdk
# Appc 1275 add events purchase intent to sdk

**What does this PR do?**

Adds a purchase_intent event to the sdk that gets fired when user launches billing flow
Changes the package name
from: com.appcoins.sdk.android.billing
to: com.appcoins.billing.sdk

**Database changed?**

No

**Where should the reviewer start?**

- [ ] EventLogger.java

**How should this be manually tested?**

 Open bds-sdk proj and run toolbox. Press launch billing flow button.
Take note of the time you did this. 10 min later you can ask @jose.pereira to check if your event showed up. Pro tip: you can force the sku value to a very specific value to ease the verification process

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-1275](https://aptoide.atlassian.net/browse/APPC-1275)

**Questions:**
None

**Code Review Checklist**
